src/Components/Todo/Todo.tsx

```diff
@@ -0,0 +1,45 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  completed: boolean,
+  title: string,
+};
+
+const Todo: React.FC<Props> = ({ completed, title }) => (
+  <li
+    className={classNames(
+      'todo',
+      { completed },
+    )}
+  >
+    <label className="todo__status-label">
+      <input
+        type="checkbox"
+        className="todo__status"
+        checked={completed}
+      />
+    </label>
+
+    <span className="todo__title">{title}</span>
+    <button type="button" className="todo__remove">×</button>
+
+    {false && (
```

Comment:
why do you need this condition? It's not a good practice to use `false` as a condition